.github/ci/generate-news-feed.js

```diff
@@ -0,0 +1,113 @@
+import { Feed } from 'feed';
+import fs from 'fs';
+
+/**
+ * @type {FeedConfig[]}
+ */
+const feeds = [
+  {
+    title: '24life Updates',
+    targetFile: `../../feed.xml`,
+    source: 'https://main--24life--hlxsites.hlx.live/articles.json',
+    siteRoot: "https://main--24life--hlxsites.hlx.live",
+    link:	"https://main--24life--hlxsites.hlx.live",
```

Comment:
In macktruck this links to this feed itself. check the rss/atom specs if that is the correct usage.  e.g. `https://www.24life.com/feed.xml`

Reply:
Thanks !  I dint know 
https://validator.w3.org/feed/docs/atom.html#requiredFeedElements:~:text=A%20feed%20should%20contain%20a%20link%20back%20to%20the%20feed%20itself